R/leafletControls.R

```diff
@@ -184,7 +184,7 @@ removeDuplicatedMapDependencies <- function(map) {
 
 
 
-rasterCheckSize <- function(x, maxpixels) {
+rasterCheckSize<- function(x, maxpixels) {
```

Comment:
Can we just leave that as it was? Has nothing to do with the terra methods

Reply:
Yup, sorry bout that! Sloppy…